Source/CarthageKit/Xcode.swift

```diff
@@ -330,6 +352,12 @@ public enum SDK: Equatable {
 		case "iphonesimulator":
 			return .success(.iPhoneSimulator)
 
+		case "watchos":
+			return .success(.watchOS)
+
+		case "watchsimulator":
+			return .success(watchSimulator)
```

Comment:
Both work (this is a `static func` so `watchSimulator` doesn’t need to be resolved with `.` or `self.`), but it’d be nice to be consistent with the other uses.